.gitignore

```diff
@@ -91,3 +91,6 @@ ENV/
 # pyenv
 .python-version
 .vscode/
+
+# Local e2e test run files
+runtime/
```

Comment:
This is getting in the way every time I run tests locally. Framework caching some json files in this folder

Reply:
That is created by LISA. you can change the directory with the --working_path parameter. I usually set it to ~/tmp/lisa

Reply:
I had this parameter in my run ,still get this

![image](https://github.com/Azure/WALinuxAgent/assets/84482346/caea7a07-31b9-44b5-b164-ec2e0ff5868f)

Reply:
oh, yes, i remember sometimes they do not honor it

try changing the cwd as well

![image](https://github.com/Azure/WALinuxAgent/assets/8194100/ead04208-023f-455e-a33b-30ace2b2c537)

Reply:
thanks, it worked. I reverted the change